schemas/neutron_diagnostic/dd_neutron_diagnostic.xsd

```diff
@@ -86,7 +86,7 @@
 				<xs:annotation>
 					<xs:documentation>Normalized poloidal flux coordinate for the reconstructed profiles, the flux being normalized to its value at the separatrix</xs:documentation>
```

Comment:
This text isn't quite right.  The normalised poloidal flux is normlised using the value at the magnetic axis and the value at the edge (is it always the separatrix?).

```suggestion
					<xs:documentation>Normalized poloidal flux coordinate for the reconstructed profiles, the flux being normalized to run from 0 at the magnetic axis to 1 at the separatrix</xs:documentation>
```